src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,17 +2,24 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    public static final int DEFAULT = -1;
+    public static final int MAX_NUMBER_OF_ELEMENTS = 12;
 
-    public int total = -1;
+    private final int[] numbers = new int[MAX_NUMBER_OF_ELEMENTS];
+
+    private int total = DEFAULT;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)